hack/aks/property-based-scheduling.md

```diff
@@ -0,0 +1,428 @@
+# Use property-based scheduling with Fleet on AKS clusters: Preview
+
+This document explains how to try out the latest Fleet feature, property-based scheduling, using AKS clusters.
+
+The new property-based scheduling experience adds great flexibility to Fleet's existing resource propagation capabilities, granting users the ability to pick clusters based on their resource usage, costs, node counts, and potentially many more features. This can be very helpful in many common multi-cluster administration scenarios, including:
```

Comment:
Change "The new property-based scheduling experience adds great flexibility..." 
to "Fleet property-based scheduling (preview) adds more flexibility..."